src/features/read/components/Question.tsx

```diff
@@ -76,32 +76,31 @@ interface OnStimulusClick {
 
 function Stimulus({ decorate, disabled, onClick, stimulus }: StimulusProps) {
   const cx = cn(
-    "d-flex flex-column justify-content-between w-50 h-100",
+    `d-flex flex-column justify-content-between w-50 h-100`,
     {
       "border border-success": decorate === "CORRECT",
       "border border-danger": decorate === "WRONG",
     },
-    {
-      [styles.stimulus]: disabled === false,
-      [styles.disabledStimulus]: disabled === true,
-    }
+    { [styles.stimulus]: true }
   );
 
   return (
-    <div
-      className={cx}
-      aria-disabled={disabled}
-      onClick={() => onClick(stimulus.sys.id)}
-      role="button"
-    >
+    <div className={cx}>
       <img
         alt={stimulus.fields.image.fields.description}
         src={stimulus.fields.image.fields.file.url}
         style={{ maxWidth: "100%", height: "auto" }}
       />
-      <Button disabled={disabled} variant="link" size="lg">
-        Pick me
-      </Button>
+      {!disabled && (
```

Comment:
Just hide the button entirely after it is used so that there is only one visible action on the screen at a time.